tests/services/test_file_mover.py

```
@@ -241,18 +246,19 @@ def test_move_with_rollback_success(mock_move, mock_verifier, source_and_dest):
 @patch("app.services.file_mover.checksum_verifier")
 @patch("app.services.file_mover._move")
 @patch("pathlib.Path.unlink")
-def test_move_with_rollback_checksum_mismatch(mock_unlink, mock_move, mock_verifier, source_and_dest):
+def test_move_with_rollback_checksum_mismatch(
+    mock_unlink, mock_move, mock_verifier, source_and_dest
```

Ruff (0.14.14)

[warning] Unused function argument: `mock_unlink`

(ARG001)